pycontw/pycon.tw
# remove prefer_time for cfp

## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies


## Description
**Describe what the change is**
Remove options for cfp -"prefer_time" from src/core/choices.py and src/core/models.py then executed the following: 
$ python manage.py makemigrations
$ python manage.py makemessages -l en_US -l zh_Hant

In addition, "prefer_time" in testing files:
1. src/proposals/tests/views/test_create.py
2. src/proposals/tests/views/test_proposal.py
have to remove also.

## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Go to 'http://localhost:8000/admin/'
2. LogIn [Information](https://github.com/pycontw/pycon.tw/blob/master/docker-compose-dev.yml)
3. Click on [專業課程提案](http://localhost:8000/admin/proposals/tutorialproposal/)
4. Check if option "偏好議程時間" is removed
5. Repeated step 3-4 for [演講提案](http://localhost:8000/admin/proposals/talkproposal/)

## Expected behavior
"偏好議程時間" is removed on each page in Zh/En.

## Related Issue
If applicable, refernce to the issue related to this pull request.
#1163


**Additional context**
Screenshot:
<img width="560" alt="image" src="https://github.com/pycontw/pycon.tw/assets/26858727/69ce4a93-c447-499d-b3f9-fae34d60fdf0">
